generic/tests/netperf.py

```diff
@@ -141,6 +144,25 @@ def mtu_set(mtu):
             process.run(host_mtu_cmd % (iface, mtu), ignore_status=False,
                         shell=True)
 
+    def tweak_tuned_profile():
+        """
+
+        Tweak configuration with truned profile
+
+        """
+
+        client_tuned_profile = params.get("client_tuned_profile")
+        server_tuned_profile = params.get("server_tuned_profile")
+        host_tuned_profile = params.get("host_tuned_profile")
```

Comment:
Better give params a default value, or use params["xxx"] to avoid get a null vaule.

Reply:
these params could be empty, use if function to see if param existed. please help a review, thanks.

Reply:
If the param is null, will do nothing? I think you expect to run these command.
Why not params.get["host_tuned_profile"]? If so, you don't need to add if judgement and if the params is null, the test should fail, or params.get("host_tuned_profile", ""tuned-adm profile virtual-host""). These two formats are common in our code.

Reply:
actullay params could be null, it depends on requirement.